src/main/java/org/usfirst/frc/team1758/robot/commands/groups/GearPushAndPull.java

```diff
@@ -1,19 +1,15 @@
 package org.usfirst.frc.team1758.robot.commands.groups;
 
-import org.usfirst.frc.team1758.robot.commands.StrafeAway;
-import org.usfirst.frc.team1758.robot.commands.StrafeForward;
-import org.usfirst.frc.team1758.robot.commands.ToggleGear;
-
-import edu.wpi.first.wpilibj.command.Command;
-
 import edu.wpi.first.wpilibj.command.CommandGroup;
 
+import org.usfirst.frc.team1758.robot.commands.MoveForward;
+import org.usfirst.frc.team1758.robot.commands.ToggleGear;
+
 public class GearPushAndPull extends CommandGroup {
```

Comment:
Can we rename this to GearDeploy? that makes more sense, considering we are not pushing and pulling the gear. We are deploying it though.